modules/jupyter-widget/src/utils.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Gets CSS from a given URL and appends it to the document head
+ * @param url URL of CSS page
+ */
+function loadCss(url) {
+  let link = document.createElement('link'); //eslint-disable-line
```

Comment:
You can add `/* global document */` at the top of this file to get rid of the eslint error